addons/hr_recruitment_survey/data/survey_demo.xml

```diff
@@ -2,7 +2,7 @@
 <odoo><data noupdate="1">
     <record id="survey_recruitment_form" model="survey.survey">
         <field name="title">Recruitment Form</field>
-        <field name="user_id" ref="base.user_admin"/>
+        <field name="user_ids" eval="[(4, ref('base.user_demo'))]"/>
```

Comment:
Why this change?

Reply:
My turn to apologize :laughing:  Probably a 'copy-paste mistake. Did not mean to steal it from Mitchell Admin. Can give him back his survey if it is more suitable.